InterfaceParser/parsers/rpc_base.py

```diff
@@ -229,6 +231,8 @@ def _parse_struct(self, element, prefix):
                 raise ParseError("Unexpected subelement '{}' in struct '{}'".format(subelement.name, params["name"]))
         params["members"] = members
 
+        self._types.pop(struct.name, None)
```

Comment:
Please add appropriate comments, why we need to remove from structs

Reply:
@YarikMamykin, please create a good commit message with the explanation of what was fixed and how

Reply:
@LitvinenkoIra fixed in https://github.com/LuxoftSDL/rpc_spec/pull/3/commits/15d6abfb7656b808e59feade4b53d51b032dde3f